tests/acceptance/blockchain.controller.acceptance.tests.ts

```diff
@@ -80,10 +76,39 @@ describe('Blockchains controller (acceptance)', () => {
 
   it('retrieves the information of a specific blockchain', async () => {
```

Comment:
Could we add test cases for other scenarios? at-least the following should be covered IMO: incorrect blockchain hash, any field from the blockchain object that can be null/empty.

Reply:
added test for incorrect hash. As for field or null empty fields are already handled on the other tests implicitly, not all the fields are used